mobile-app/app/screens/WalletNavigator/screens/components/OnboardingCarousel.tsx

```diff
@@ -53,6 +55,13 @@ const slides: JSX.Element[] = [
     subtitle="Access financial opportunities with dTokens minted through decentralized vaults."
     title="Decentralized loans"
   />,
+  <ImageSlide
+    imageDark={ImageEDark}
+    imageLight={ImageELight}
+    key={4}
+    subtitle="Seamless transfers between DeFiChain and MetaChain via generated EVM address."
+    title="EVM compatible"
+  />,
 ];
```

Comment:
Right side of the image is a little bit cropped
<img width="767" alt="image" src="https://github.com/BirthdayResearch/defichain-wallet/assets/17063281/944d37b8-c32f-4036-af21-ce4e43603123">